Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM python:3
+
+ENV PYTHONDONTWRITEBYTECODE 1
+ENV PYTHONUNBUFFERED 1
```

Comment:
just a tip for dockerfiles, you usually want to combine adjacent `ENV` statements so they don't create unnecessary extra layers, in this case you'd write

```dockerfile
ENV PYTHONDONTWRITEBYTECODE=1 PYTHONUNBUFFERED=1
``` 

or if you don't like it all one line you can write it with some backslashes

```dockerfile
ENV \
    PYTHONDONTWRITEBYTECODE=1 \
    PYTHONUNBUFFERED=1
```